nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/resources/nifi-web-api-context.xml

```diff
@@ -114,7 +123,23 @@
         <property name="keyService" ref="keyService"/>
         <property name="dtoFactory" ref="dtoFactory"/>
     </bean>
+    <bean id="authorizableLookup" class="org.apache.nifi.web.StandardAuthorizableLookup">
+        <property name="controllerFacade" ref="controllerFacade"/>
+        <property name="processorDAO" ref="processorDAO"/>
+        <property name="inputPortDAO" ref="inputPortDAO"/>
+        <property name="outputPortDAO" ref="outputPortDAO"/>
+        <property name="processGroupDAO" ref="processGroupDAO"/>
+        <property name="remoteProcessGroupDAO" ref="remoteProcessGroupDAO"/>
+        <property name="labelDAO" ref="labelDAO"/>
+        <property name="funnelDAO" ref="funnelDAO"/>
+        <property name="connectionDAO" ref="connectionDAO"/>
+        <property name="controllerServiceDAO" ref="controllerServiceDAO"/>
+        <property name="reportingTaskDAO" ref="reportingTaskDAO"/>
+        <property name="templateDAO" ref="templateDAO"/>
+        <property name="snippetDAO" ref="snippetDAO"/>
+    </bean>
     <bean id="serviceFacade" class="org.apache.nifi.web.StandardNiFiServiceFacade">
+        <constructor-arg ref="authorizableLookup" />
```

Comment:
I have no issue with using constructor arguments. However, why are we only using it on that one property?